local/network.go

```diff
@@ -825,7 +827,8 @@ func (ln *localNetwork) restartNode(
 
 	if binaryPath != "" {
 		nodeConfig.BinaryPath = binaryPath
-		nodeConfig.Flags[config.BuildDirKey] = filepath.Dir(binaryPath)
+		// remove old value if present, to look for vm binaries in new binary path location
+		delete(nodeConfig.Flags, PluginDirKey)
```

Comment:
This seems pretty jank can't the `Flags` be defined by the user? We should definitely be keeping this override if the user requests it... right?

Reply:
Yup flags were possibly defined by the user when first created the node. In this case I assumed the user trying to restart a node by specifying a new binary (usually a different avago version in tests), was expecting also for the plugin dir to automatically change. I made this as I realised restart was lacking plugin dir param available. The other and probably cleaner option would be to add that plugin dir param to this function.

Reply:
Added the extra plugin param to RestartNode. Also to AddNode.